test/functional/Java8andUp/playlist.xml

```diff
@@ -625,6 +625,57 @@
 		</subsets>
 	</test>
 
+	<test>
+		<testCaseName>gcPolicyNogcTest</testCaseName>
+		<variations>
+			<variation>-Xgcpolicy:nogc -Xms128m -Xmx1G -verbose:gc</variation>
+			<variation>-XX:+UseNoGC -Xms128m -Xmx1G</variation>
+		</variations>
+		<command>$(JAVA_COMMAND) $(JVM_OPTIONS) \
+	-cp $(Q)$(RESOURCES_DIR)$(P)$(TESTNG)$(P)$(TEST_RESROOT)$(D)GeneralTest.jar$(Q) \
+	org.testng.TestNG -d $(REPORTDIR) $(Q)$(TEST_RESROOT)$(D)testng.xml$(Q) -testnames gcPolicyNogcTest \
+	-groups $(TEST_GROUP) \
+	-excludegroups $(DEFAULT_EXCLUDE); \
+	$(TEST_STATUS)</command>
+		<levels>
+			<level>extended</level>
+		</levels>
```

Comment:
We should also add an `<impls>
			<impl>openj9</impl>
		</impls>` tag and add openj9 and ibm as impl values, example of use is [here]( https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/systemtest/playlist.xml#L349).